### PR TITLE
[sdks] Add a community SDKs page

### PR DESCRIPTION
**Pull request by gregnazario**
### Description
Moves Swift SDK and all future SDKs to a community page.  This allows people to add SDKs that they want known, but don't necessarily have the vetting associated with future specification work.

### Checklist

- Do all Lints pass?
  - [x] Have you ran `pnpm spellcheck`?
  - [x] Have you ran `pnpm fmt`?
  - [x] Have you ran `pnpm lint`?

### CODE REVIEW

File: apps/nextra/pages/en/build/sdks/community-sdks.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+# Community SDKs
+
+Here is a list of community built SDKs for Aptos.  These may not be fully vetted by the Aptos team, and may still be in development.
+They are still provided as a resource for all developers.
+
+## SDKs
+
+- [Swift SDK](./community-sdks/swift-sdk)
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/community-sdks.mdx:12]: Can we make this a `<Card />`?

File: apps/nextra/pages/en/build/sdks/community-sdks.mdx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+{/* TODO Move Kotlin SDK here */}
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks/community-sdks.mdx:14]: Should we include the Kotlin SDK move in this PR or do you wanna do it in a follow up?

Comment (gregnazario) [reply apps/nextra/pages/en/build/sdks/community-sdks.mdx:14]: I want to follow up after the Kotlin SDK PR is closed.  No reason to make him refactor his PR for now.

File: apps/nextra/pages/en/build/sdks.mdx [hunk 3]
```diff
@@ -64,6 +64,14 @@ Use these Aptos software development kits (SDKs), in combination with the [Aptos
...
   </div>
+  </Card>
+  <Card href="./sdks/community-sdks" className="flex flex-row gap-4">
+  <div className="flex flex-col gap-2">
+    <Card.Title>Community SDKs</Card.Title>
+    <Card.Description>
+      Community built SDKs
+    </Card.Description>
+  </div>
 </Card>
```
Comment (hariria) [inline apps/nextra/pages/en/build/sdks.mdx:75]: Fix Tabs

**Review by hariria — APPROVED** [review]

**Pull request merged**